frontend/apps/reader/modules/readertypeset.lua

```diff
@@ -307,11 +312,36 @@ This stylesheet is to be used only with FB2 and FB3 documents, which are not cla
             end
             return text
         end,
-        sub_item_table = obsoleted_table,
         checked_func = function()
             return obsoleted_css[self.css] ~= nil
-        end
+        end,
+        sub_item_table = obsoleted_table,
+        separator = true,
     })
+    if self.ui.document.is_txt then
```

Comment:
Thank you for that - and not showing this hard to explain item to people who don't read TXT files.
Might be worth having a `help_text` though. Quick proposal (possibly not accurate):

When enabled, the rendering engine will use heuristics (text wrapped under some max line length, single or double blank lines marking paragraphs, indentation or centered text considered heading...) to guess how to better render the text.
When disabled, the text is rendered as is in a monospace font, which may be best if it has some ASCII art drawing or format things with spaces.